lib/ts/tests.cc

```diff
@@ -19,7 +19,7 @@
   limitations under the License.
 */
 
-#include "ts/TestBox.h"
+#include "ts/Regression.h"
```

Comment:
Not really, but I noticed it while fixing another problem. I can file a separate Jira if you feel strongly, but I'm certain it's the right fix (without it, we get compiler warnings).